src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: 200px;
+  justify-content: center;
+  align-content: center;
```

Comment:
Gap between cards should be 48px (in all cases)
![image](https://user-images.githubusercontent.com/115103012/201905637-093347ad-ebe6-4aae-923d-e4bc2ca51939.png)

Reply:
fixed